contracts/test/foundry/FleekERC721/Mint.t.sol

```diff
@@ -49,4 +49,22 @@ contract Test_FleekERC721_Mint is Test_FleekERC721_Base {
 
         assertEq(CuT.balanceOf(deployer), 1);
     }
+
+    function testFuzz_mint(
+        address to,
+        string memory appName,
+        string memory description,
+        string memory externalURL,
+        string memory ens,
+        string memory commitHash,
+        string memory gitRepository,
+        string memory logo,
+        uint24 color
+    ) public {
+        vm.assume(to != address(0));
```

Comment:
This makes me think, should we prevent mints that try to mint to addresss(0) on the contract?